vendor/github.com/hashicorp/terraform/helper/structure/flatten_json.go

```diff
@@ -0,0 +1,16 @@
+package structure
```

Comment:
Why terraform dependency was updated?

Reply:
Honestly I don't know probably the version in dependencies of Terraform itself changed.

Reply:
Please verify, cause harshicorp still will build using `provider` folder dependencies

Reply:
In fact this does not have any change in PR with current master. So either it was pulled into master with another PR or no longer here.